Archives/VolFile.cpp

```diff
@@ -214,49 +212,38 @@ namespace Archives
 		// Allowing duplicate names when packing may cause unintended results during binary search and file extraction.
 		CheckSortedContainerForDuplicateNames(volInfo.internalNames);
 
+		volInfo.stringTableLength = 0;
```

Comment:
This seems like it would be better placed in `PrepareHeader`.

Reply:
I'll move it in there. The PrepareHeader function needs to be broken into smaller chunks at some point but I'm having trouble figuring out a good way to break it up since all the calculations are done sort of intermingled.